flink-runtime/src/test/java/org/apache/flink/runtime/rest/RestServerEndpointITCase.java

```diff
@@ -340,60 +331,52 @@ public void testBadHandlerRequest() throws Exception {
         } catch (ExecutionException ee) {
             Throwable t = ExceptionUtils.stripExecutionException(ee);
 
-            assertTrue(t instanceof RestClientException);
+            assertThat(t instanceof RestClientException).isTrue();
 
             RestClientException rce = (RestClientException) t;
 
-            assertEquals(HttpResponseStatus.BAD_REQUEST, rce.getHttpResponseStatus());
+            assertThat(rce.getHttpResponseStatus()).isEqualTo(HttpResponseStatus.BAD_REQUEST);
```

Comment:
Maybe we can use `assertThatFuture` as well.

Reply:
@Jiabao-Sun if we use `assertThatFuture`, how can we do the test for `rce.getHttpResponseStatus()` witch is not a msg